test/e2e/tests/request-queuing/multiple-networks-dapps-txs.spec.js

```diff
@@ -109,15 +109,13 @@ describe('Request Queuing for Multiple Dapps and Txs on different networks.', fu
 
         // Dapp one send tx
         await driver.switchToWindowWithUrl(DAPP_URL);
-        await driver.executeScript(`window.location.reload()`);
```

Comment:
its no longer required to reload the page after connecting to ensure that the proxies are correctly synced